docs/3.api/4.advanced/1.hooks.md

```diff
@@ -33,7 +33,51 @@ Hook                   | Arguments           | Environment     | Description
 
 Check the [schema source code](https://github.com/nuxt/nuxt/blob/main/packages/schema/src/types/hooks.ts#L53) for all available hooks.
 
-:NeedContribution
+Hook                     | Arguments                  | Environment     | Description
```

Comment:
I think we can lose the environment column as they all happen within the build server.

Later, we can refactor to perhaps list these in the normal order they are called, but I wouldn't want to block the PR for that.